contrib/babelfishpg_tsql/src/guc.c

```diff
@@ -771,6 +783,15 @@ define_custom_variables(void)
 							   GUC_NOT_IN_SAMPLE | GUC_NO_RESET_ALL,
 							   NULL, NULL, NULL);
 
+	DefineCustomStringVariable("babelfishpg_tsql.psql_logical_babelfish_db_name",
+							   gettext_noop("Sets a Babelfish database name"),
+							   NULL,
+							   &pltsql_psql_logical_babelfish_db_name,
+							   NULL,
+							   PGC_SUSET,
+							   GUC_NOT_IN_SAMPLE | GUC_NO_RESET_ALL,
```

Comment:
We should add `GUC_DISALLOW_IN_FILE | GUC_DISALLOW_IN_AUTO_FILE` since we don't want to allow setting it  in conf file.